sections/main-collection-banner.liquid

```diff
@@ -2,14 +2,25 @@
 {{ 'component-collection-hero.css' | asset_url | stylesheet_tag }}
 
 {%- style -%}
+  .section-{{ section.id }}-padding {
+    padding-top: {{ section.settings.padding_top | times: 0.75 | round: 0 }}px;
+    padding-bottom: {{ section.settings.padding_bottom | times: 0.75 | round: 0 }}px;
+  }
+
+  @media screen and (min-width: 750px) {
+    .section-{{ section.id }}-padding {
+      padding-top: {{ section.settings.padding_top }}px;
+      padding-bottom: {{ section.settings.padding_bottom }}px;
+    }
+  }
```

Comment:
This is looking good, thanks @kjellr 

I do have a question (_and there's a good chance I'm missing some context on whether this was intentional_) -- when I enable **Collection banner** > **Show collection image**, the section padding works on mobile (`max-width: 749px`) but not on desktop:

<details>
  <summary>See video</summary>

  https://user-images.githubusercontent.com/28404165/173890520-dc53a070-966d-4ae1-94a9-94be4a68d319.mp4

</details>

Reply:
To reduce the noise in this PR, I created a draft PR (with store editor link) for the above: https://github.com/Shopify/dawn/pull/1808

Feel free to comment on the draft PR 👍🏼